webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/internal/SharedFilesAPI.scala

```diff
@@ -65,13 +67,18 @@ import net.liftweb.json.JsonAST.JNull
 import net.liftweb.json.JsonAST.JObject
 import net.liftweb.json.JsonAST.JString
 import net.liftweb.json.JsonAST.JValue
+import org.apache.commons.fileupload.FileUploadBase.SizeLimitExceededException
 import org.joda.time.DateTime
 import org.joda.time.Instant
 import scala.jdk.CollectionConverters.*
+import scala.util.Try
 import zio.ZIO
 import zio.syntax.*
 
 object SharedFilesAPI {
+  // Configured max number of bytes in an uploaded file in Lift
```

Comment:
the comment need to talk about `LiftRules.maxMimeSize` and `LiftRules.maxMimeFileSize`